src/modules/transaction/components/TransactionForm.scss

```diff
@@ -67,54 +68,40 @@
         padding-left: 1rem;
         padding-right: 1rem;
         background: $brand-red;
-        line-height: 2rem;
         font-weight: 200;
     }
 
+    .buttonContainer {
+        flex-direction: row;
+    }
+
     .imageButton {
-        width: 100%;
+        width: 48%;
         height: 2.5rem;
-        margin-top: 1.2rem;
         color: $brand-grey-medium;
         background: white;
         border: solid 1px $brand-grey;
+        float: left;
         display: flex;
         align-items: center;
-        margin-bottom: auto;
-
-        img {
-            margin-left: 2rem;
-            width: 1.2rem;
-            height: 1.2rem;
-        }
 
         p {
-            margin-left: -3.2rem; //Width of image + margins
             text-align: center;
             width: 100%;
             font-size: 0.8rem;
         }
     }
 
     .kudoButton {
-        width: 100%;
+        width: 48%;
         height: 2.5rem;
-        margin-top: 3rem;
         color: $brand-white;
         background: $brand-green;
         border: solid 1px $brand-grey;
-        display: flex;
+        float: right;
         align-items: center;
```

Comment:
Also here, remove the floats and use flexbox. `align-items` wouldn't do much without `display: flex`.